lib/commands/publish.js

```diff
@@ -131,6 +131,13 @@ class Publish extends BaseCommand {
 
     const resolved = npa.resolve(manifest.name, manifest.version)
 
+    const latestDistTag = await this.#latestPublishedVersion(resolved)
+    const latestTagIsGreater = latestDistTag ? semver.gte(latestDistTag, manifest.version) : false
```

Comment:
```suggestion
    const latestVersion = await this.#latestPublishedVersion(resolved)
    const latestTagIsGreater = !!latestVersion && semver.gte(latestVersion, manifest.version)
```
fixing the variable name